target/linux/ramips/dts/mt7621_tplink_er605-v2.dts

```diff
@@ -11,7 +11,9 @@
 	compatible = "tplink,er605-v2", "mediatek,mt7621-soc";
 
 	chosen {
-		bootargs = "console=ttyS0,115200 noinitrd";
+		// Override bootargs because u-boot passes wrong root parameter.
+		// Instead allow kernel determine root automatically by looking for rootfs volume
+		bootargs-override = "console=ttyS0,115200 noinitrd ubi.mtd=3,2048";
```

Comment:
Please also keep `bootargs` to override the default value in the `mt7621.dtsi`. https://github.com/openwrt/openwrt/commit/6361a952afa4827eb5e9d5221b9bb4b54a2019d5
https://github.com/openwrt/openwrt/blob/9ac377d0e012e42d9cd69ccc6471fae93c738c1d/target/linux/ramips/dts/mt7621.dtsi#L41
```
bootargs-override = "console=ttyS0,115200 noinitrd ubi.mtd=3,2048";
```
Although this will not pass to the kernel, it can make the device tree more robust.

Reply:
Fixed, thanks!